### PR TITLE
[docker-ptf]: Add teamd dependency to ptf

### PR DESCRIPTION
**Pull request by Pterosaur**
Signed-off-by: Ze Gan <ganze718@gmail.com>

#### Why I did it
Teamd may be used in PTF to test portchannel case

#### How I did it
1. Install 3rd packages needed by teamd in Dockerfile.j2 
2. Add teamd module and its relative modules in docker-ptf.mk

#### How to verify it
Execute `which teamd` in ptf container should got the following result.
```bash
root@72db4f0d2eec:/# which teamd
/usr/bin/teamd
```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [x] 201911
- [x] 202006
- [x] 202012

### CODE REVIEW

File: platform/vs/docker-ptf.mk [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 DOCKER_PTF = docker-ptf.gz
 $(DOCKER_PTF)_PATH = $(DOCKERS_PATH)/docker-ptf
-$(DOCKER_PTF)_DEPENDS += $(LIBTHRIFT) $(PYTHON_THRIFT) $(PTF) $(PYTHON_SAITHRIFT)
+$(DOCKER_PTF)_DEPENDS += $(LIBTHRIFT) $(PYTHON_THRIFT) $(PTF) $(PYTHON_SAITHRIFT) $(SWSS) $(LIBTEAMDCT) $(LIBTEAM_UTILS)
```
Comment (qiluo-msft) [inline platform/vs/docker-ptf.mk:5]: >SWSS [](start = 84, length = 4)

Why need swss?

Comment (qiluo-msft) [inline platform/vs/docker-ptf.mk:5]: >LIBTEAMDCT) $(LIBTEAM_UTILS [](start = 92, length = 27)

Could you use debian vanilla packages?
No need to install their dependencies explicitly.

Comment (Pterosaur) [reply platform/vs/docker-ptf.mk]: I have confirmed that the teamd from vanilla package can be used.

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
Blocked by another fix https://github.com/Azure/sonic-buildimage/pull/6999

**Pull request merged**